### DISCUSSION TITLE
`/.well-known/change-password`

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#1826] [Ideas & RFCs]
@siddhantk232 looks like this is a standard URL to change password. Maybe we update our URL to use this URL instead. Other option is we redirect, but if some standard says change password should be on this URL, why not just follow it.

<img width="832" alt="image" src="https://github.com/fastn-stack/fastn/assets/58662/5b08f18b-05dc-4278-b404-f0dc30fdf989">

**Comment by siddhantk232** [comment]
good catch! I was able to find https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml which lists `.well-known/change-password/`. I'll see how we can incorporate this with our password reset feature.
